src/views/ForgotPasswordPageSend/index.js

```diff
@@ -42,38 +46,40 @@ export default class ForgotPasswordPageSend extends Component {
         Username: this.state.email,
       };
 
-      cognitoIdentityServiceProvider.forgotPassword(params, function(err, data) {
-        if (err) {
-          if (err.code === "LimitExceededException") {
-            this.setState({
-              err: "tooManyAttempts"
+      cognitoIdentityServiceProvider.forgotPassword(
+        params,
+        function (err, data) {
+          console.log("HERE!");
+          console.log(err);
+          console.log(data);
```

Comment:
I think you forgot to remove these lol

Reply:
Yup lol 😅 This function gets moved to a different file in my next PR tho, so I got rid of that there!